src/AcceptanceTests/AcceptanceTests.csproj

```diff
@@ -23,4 +23,23 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This will also cut down the test execution time quite a bit. We did this for several persisters already. It doesn't make sense to execute all the acceptance tests